Documentation/DataSources/DeprecatedWorkbookRetrievalTool.md

```diff
@@ -0,0 +1,352 @@
+# Deprecated Workbook Retrieval Tool
+
+
+Private and Legacy workbooks have been deprecated and are unaccessable from the Azure Portal UX. If you are here because you are looking for the deprecated workbook that you forgot to convert before the deadline, you're in luck! We've created a tool that will allow you to retrieve the content of your old workbook and load it into a new workbook so that you can save it as a new workbook.
+
+_To move forward with the deprecation of these workbooks, this tool will only be available for a limited time until **August 31, 2021**._
+
+## Private Workbook Retrieval
+1. Open up an new or empty workbook
+2. Go into Edit mode in the toolbar and navigate to the advanced editor
+    ![advanced editor](../Images/DeprecatedWb_RetrievalTool_AdvancedEditor.png)
+3. Copy the following workbook json and paste it into your open advanced editor
+>{
```

Comment:
this is formatted as a quote, so it looks like text

it should be code, between ``` blocks to make it look like code and format nicely